recipes/r8brain-free-src/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class R8brainFreeSrcConan(ConanFile):
+    name = "r8brain-free-src"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/avaneev/r8brain-free-src"
+    description = "High-quality pro audio sample rate converter / resampler C++ library"
+    topics = ("audio", "sample-rate", "conversion", "audio-processing", "resampler")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            raise ConanInvalidConfiguration("Shared r8brain-free-src cannot be built with Visual Studio")
```

Comment:
Adding [`CMAKE_EXPORT_WINDOWS_SYMBOLS`](https://cmake.org/cmake/help/v3.4/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html) doesn't work?

Reply:
see above the comments:

[Click](https://github.com/conan-io/conan-center-index/pull/2536#issuecomment-674457138)